hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -29,12 +54,41 @@ def lambda_handler(event, context):
     an action message on the actions queue and here's where they're popped
     off and dealt with.
     """
+    lambda_init_once()
     for sqs_record in event["Records"]:
         # TODO research max # sqs records / lambda_handler invocation
-        action_message = ActionMessage.from_aws_message(json.loads(sqs_record["body"]))
+        action_message = ActionMessage.from_aws_message(sqs_record["body"])
 
         logger.info("Performing action: action_message = %s", action_message)
 
         perform_label_action(action_message, action_message.action_label)
 
     return {"action_performed": "true"}
+
+
+if __name__ == "__main__":
+    lambda_init_once()
+
+    banked_signals = [
+        BankedSignal("2862392437204724", "bank 4", "te"),
+        BankedSignal("4194946153908639", "bank 4", "te"),
```

Comment:
should the bank names be different?

Reply:
Never read? I think two matches in the same bank is possible

Reply:
bank == dataset? or is bank a new concept?